src/quo2/components/list_items/account/view.cljs

```diff
@@ -0,0 +1,138 @@
+    (fn [{:keys [type title-icon? selectable? blur? account-props customization-color token-props
+                 balance-props
+                 on-press
+                 on-options-press
+                 theme]
+          :or   {customization-color :blue
+                 type                :default
+                 blur?               false
+                 title-icon?         false}}]
+      (let [on-press-out (fn []
+                           (let [new-state (if @active-or-selected?
+                                             :default
+                                             (if (and (= type :default) selectable?)
+                                               :selected
+                                               :active))]
+                             (when @timer (js/clearTimeout @timer))
+                             (reset! timer nil)
+                             (reset! active-or-selected? (or (= new-state :active)
+                                                             (= new-state :selected)))
+                             (reset! state new-state)
+                             (when on-press
+                               (on-press))))]
+        (rn/use-effect
+         #(cond (and selectable? (= type :default) (= @state :active))         (reset! state :selected)
+                (and (not selectable?) (= type :default) (= @state :selected)) (reset! state :active))
+         [selectable?])
+        [rn/pressable
+         {:style               (style/container
+                                {:state @state :blur? blur? :customization-color customization-color})
+          :on-press-in         on-press-in
+          :on-press-out        on-press-out
+          :accessibility-label :container}
+         [rn/view {:style style/left-container}
+          [account-avatar/view (assoc account-props :size 32)]
+          [rn/view {:style style/account-container}
+           [rn/view
+            {:style style/account-title-container}
+            [text/text
+             {:weight :semi-bold
+              :size   :paragraph-1}
+             (:name account-props)]
+            (when title-icon?
+              [rn/view
+               {:style               style/keycard-icon-container
+                :accessibility-label :keycard-icon}
+               [icon/icon :i/keycard
+                {:size  20
+                 :color (if blur?
+                          colors/white-opa-40
+                          (colors/theme-colors colors/neutral-50 colors/neutral-40 theme))}]])]
+           [text/text {:size :paragraph-2}
+            [text/text
+             {:size   :paragraph-2
+              :weight :monospace
+              :style  (style/account-address blur? theme)}
+             (:address account-props)]]]]
+         [rn/view {:style (when (= type :tag) style/token-tag-container)}
+          (when (or (= type :balance-neutral)
```

Comment:
another nit, but could be easier to read if split into smaller components for the conditions, but it could be that it's just my personal preference 👀

Reply:
Actually that makes lot of sense to improve readability, just updated the code with your suggestion :)